librz/analysis/p/analysis_bf.c

```diff
@@ -36,6 +36,8 @@ static void bf_syscall_read(RzILVM *vm, RzILOpEffect *op) {
 	rz_bv_free(bv);
 }
 
+static PrintfCallback cb_printf = NULL;
```

Comment:
i really do not like this.

Reply:
it is decreed that all rizin output must go through rz_cons but i see your comment below